### ISSUE TITLE
Website does not rebuild

### ISSUE THREAD

**Issue by colinodell** [philsturgeon/phpversions.info#623]
The old Travis CI build process doesn't seem to be working, so updates made to this repository are not getting reflected on the website.

**Comment by WanWizard** [comment]
Is anyone still maintaining this? 

I was looking add adding PHP 8.1, and noticed there is a lot of outdated info online?

**Comment by CdrMarks** [comment]
I too am interested if this is still being maintained.

**Comment by dzuelke** [comment]
@colinodell would it make sense to retire the site instead? unsure if it's really that useful anymore to people, and wrong/outdated info IMO is much worse than no info...

**Comment by colinodell** [comment]
Yeah, I agree.

@matthewtrask were you ever able to figure out who controls the DNS?

**Comment by dzuelke** [comment]
Is it @philsturgeon maybe?

**Comment by matthewtrask** [comment]
@colinodell @dzuelke no I dont know who has the DNS. I would imagine whoever Phil bought the domain from has the DNS records to match the CNAME in the root of the project. 

I just looked at the domains I have and I have phpversions.org. I can't remember why I bought it but if we want this to keep living on, we could move the project there?

**Comment by philsturgeon** [comment]
Not me for sure, sorry. I did ask a few years ago but nobody knew. https://twitter.com/philsturgeon/status/1008097683117105153?s=20&t=d3XxLBuyX8X8dABLsS_ZrA

**Comment by matthewtrask** [comment]
@dzuelke I agree. This site has had a good life, but the landscape has changed so it may be best to let this site go quietly into the night.

cc @colinodell

**Comment by colinodell** [comment]
Agreed, it is time.

**Comment by WanWizard** [comment]
I doubt the people Phil originally targeted with the website are now using containers and buildpacks. Those that do don't need the info on the website. 

For us as a SME managed hoster, it was a showcase to our customers, both for the variaty of versions we support, but also that the versions we used were up to date.

But I agree, better no website than a non-functioning one.

**Comment by dzuelke** [comment]
WDYAT about having the website redirect to this repo, and archiving the repo?

That might be the simplest solution overall.

**Comment by colinodell** [comment]
Good idea! I've manually replaced the site with a simple message linking back here, and I'll archive this repo momentarily.

On behalf of everyone who helped maintain this site over the years, I'd like to thank everyone who made contributions and helped drive the adoption of modern PHP versions!  You make the PHP community the amazing place it is today 💜🐘
